qcfractal/cli/qcfractal_server.py

```diff
@@ -149,10 +161,28 @@ def parse_args():
     user_remove = user_subparsers.add_parser("remove", help="Remove a user.")
     user_remove.add_argument("username", default=None, type=str, help="The username to remove.")
 
-    # Dsahboard
+    # Dashboard
     dashboard = subparsers.add_parser("dashboard", help="Launches a Dashboard for the server (beta).")
     dashboard.add_argument("--base-folder", **FractalConfig.help_info("base_folder"))
 
+    # Backup
+    backup = subparsers.add_parser("backup", help="Creates a postgres backup file of the current database.")
+    backup.add_argument(
+        "--filename",
+        default=None,
+        type=str,
+        help="The filename to dump the backup to, defaults to 'database_name.bak'.",
+    )
+    backup.add_argument("--base-folder", **FractalConfig.help_info("base_folder"))
+
+    # Restore
+    restore = subparsers.add_parser("restore", help="Restores the database from a backup file.")
+    restore.add_argument("filename", default=None, type=str, help="The filename to restore from.")
+    restore.add_argument("--base-folder", **FractalConfig.help_info("base_folder"))
+    restore.add_argument(
+        "--force", action="store_true", help="If True, do not ask if the user wishes to delete the current database."
```

Comment:
Since we are using `--overwrite-config` to overwrite the installation, would it be better to also use `--overwrite-db` or just `--overwrite` here?
The word overwrite has no ambiguity.

Reply:
Since we always overwrite the `db` in the current setup I differentiated this with the word `--force` here to skip the check. This was mostly added so that I could quickly tests things and should perhaps be removed at this point. Thoughts?

Reply:
I have removed this option for now.